src/localizer.cpp

```diff
@@ -29,7 +29,10 @@ QMap<QString, StaticLanguage> s_languageMap{
     {"co", StaticLanguage{"Corsu", ""}},
     {"es_AR", StaticLanguage{"Spanish (Argentina)", "Español, Argentina"}},
     {"es_MX", StaticLanguage{"Spanish (Mexico)", "Español, México"}},
+    {"en_GB", StaticLanguage{"English (United Kingdom)", ""}},
```

Comment:
Maybe just UK, to avoid the language getting too long?

Reply:
We have space enough because the language is in a separate line.
![image](https://user-images.githubusercontent.com/515957/112840034-2501b200-909f-11eb-8fed-ba23f7ba71d9.png)